packages/pharos/src/components/popover/PharosPopover.react.stories.jsx

```diff
@@ -0,0 +1,118 @@
+import { PharosPopover } from '../../react-components/popover/pharos-popover';
+import { PharosButton } from '../../react-components/button/pharos-button';
+import { configureDocsPage } from '@config/docsPageConfig';
+import { PharosContext } from '../../utils/PharosContext';
+
+export default {
+  title: 'Components/Popover',
+  component: PharosPopover,
+  decorators: [
+    (Story) => (
+      <PharosContext.Provider value={{ prefix: 'storybook' }}>
+        <Story />
+      </PharosContext.Provider>
+    ),
+  ],
+  parameters: {
+    docs: {
+      page: configureDocsPage('popover'),
+    },
+  },
+};
+
+export const Base = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover">
+        <div style="padding: 1rem">Some very simple contents</div>
+      </PharosPopover>
+    </div>
+  ),
+};
+
+export const Events = {
+  render: () => (
+    <div>
+      <PharosButton id="my-button" data-popover-id="my-popover" icon-right="chevron-down">
+        Click Me
+      </PharosButton>
+      <PharosPopover id="my-popover">
+        <div style="padding: 1rem;display: flex; flex-direction: column; gap: 1rem;">
+          <div style="padding: 1rem">Some very simple contents</div>
+          <PharosButton
+            onClick={() => {
+              const menu = document.querySelector('storybook-pharos-popover');
+              menu.open = false;
+            }}
+          >
+            Close
+          </PharosButton>
+        </div>
+      </PharosPopover>
+    </div>
+  ),
+};
+
+export const OnBackground = {
+  render: () => (
+    <div style="background: #333333; height: 500px;">
```

Comment:
I think you can tell Storybook to use a dark background for a single story; haven't looked at the details in a minute but worth a quick peek.

Reply:
https://github.com/storybookjs/storybook/issues/8570#issuecomment-1196688341

This worked for me 👍